price-feeder/CHANGELOG.md

```diff
@@ -59,6 +59,7 @@ This was released as a part of [Umee Prop 27.](https://www.mintscan.io/umee/prop
 ### Bugs
 
 - [1428](https://github.com/umee-network/umee/pull/1428) Update umeed version to an actual tag.
+- [1615](https://github.com/umee-network/umee/pull/1615) Parse multiple candles from OsmosisV2 response
```

Comment:
This needs to go under the `## [Unreleased]` section - likely will be a part of v2.0.1

Reply:
Whoops, thought it was. Good catch.

Reply:
https://github.com/umee-network/umee/pull/1615/commits/ee027163f9df27eb8a5b6e137e4261764885564f